src/open_inwoner/accounts/models.py

```diff
@@ -219,44 +222,86 @@ class Meta:
         verbose_name = _("User")
         verbose_name_plural = _("Users")
 
+        constraints = [
```

Comment:
Feel free to continue with this next week. I like these more flexible unique constraints, having them depending on the login time makes sense in my eyes.